sae/http.go

```diff
@@ -10,13 +10,19 @@ import (
 
 // CreateHandlers returns all VM-specific HTTP handlers to be exposed by the
 // node, keyed by extension.
-func (vm *VM) CreateHandlers(context.Context) (map[string]http.Handler, error) {
-	return nil, errUnimplemented
+func (vm *VM) CreateHandlers(ctx context.Context) (map[string]http.Handler, error) {
+	s, err := vm.NewHTTPHandler(ctx)
+	if err != nil {
+		return nil, err
+	}
+	return map[string]http.Handler{
+		"/": s,
```

Comment:
In subnet-evm and coreth we route the simple HTTP traffic to `/rpc` and the websocket traffic to `/ws`. We should do that here as well imo